dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -1144,11 +1144,18 @@ public void removeContentFromLiveIndex(final Contentlet content) throws DotHiber
     public void removeContentFromIndexByStructureInode(final String structureInode)
             throws DotDataException, DotSecurityException {
         final ContentType contentType = APILocator.getContentTypeAPI(
-                APILocator.getUserAPI().getSystemUser()).find(structureInode);
+                APILocator.systemUser()).find(structureInode);
         if (contentType == null) {
             throw new DotDataException(
                     "ContentType with Inode or VarName: " + structureInode + "not found");
         }
+        removeContentFromIndexByContentType(contentType);
+    }
+
+
+    public void removeContentFromIndexByContentType(final ContentType contentType)
```

Comment:
mark me as override

Reply:
do we need db annotation here?

Reply:
it was in the original version. I just overloaded the method and kept the annotation

Reply:
The annotation is placed on the overloaded version that actually hots the db